### PR TITLE
Regen all APIs

### PR DESCRIPTION
**Pull request by saicheems**

### CODE REVIEW

**Comment by saicheems** [comment]
1. Ran `npm run generate-apis`
2. Bumped the version in `package.json` to `19.1.0`
3. Added notes to `CHANGELOG.md`

Let me know if I need to do anything else.

**Comment by saicheems** [comment]
Note that the build failure is due to `dataproc:v1alpha1`, which is returning a 404 for it's discovery doc. It's not actually an issue with the library.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,19 @@
+##### 19.1.0 - 27 June 2017
```
Comment (jmdobry) [inline CHANGELOG.md]: Please bump to 20.0.0 instead, as there are breaking changes. Thanks!

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "googleapis",
-  "version": "19.0.0",
+  "version": "19.1.0",
```
Comment (jmdobry) [inline package.json]: Please bump to 20.0.0 instead, as there are breaking changes. Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (saicheems) [inline CHANGELOG.md]: Ah, right - done!

File: package.json (hunk 2 above)
Comment (saicheems) [inline package.json]: Done

**Pull request merged**